### PR TITLE
values: add a docstring to MultiProcessValue

### PR DESCRIPTION
**Pull request by xavfernandez**
To document the method as semi-supported.

Adresses #509

### CODE REVIEW

File: prometheus_client/values.py [hunk 1]
```diff
@@ -28,10 +28,17 @@ def get(self):
             return self._value
 
 
-def MultiProcessValue(_pidFunc=os.getpid):
+def MultiProcessValue(process_identifier=os.getpid):
+    """Returns a MmapedValue class based on a process_identifier function.
+
+    The 'process_identifier' function MUST comply with this simple rule:
```
Comment (brian-brazil) [inline prometheus_client/values.py]: Don't mix must and should.

Comment (xavfernandez) [reply prometheus_client/values.py]: Updated

**Comment by brian-brazil** [comment]
Thanks!

**Comment by tyler-8** [comment]
By the way, this contained a breaking change in some cases as it wasn't just a docstring update technically. `process_identifier` used to be named `_pidFunc` so for anyone passing that as an explicit kwarg will run in to errors.

**Comment by brian-brazil** [comment]
The Python convention is that names beginning with an underscore are internal, thus changing or removing them is not breaking.

**Pull request merged**